kinode/packages/app_store/app_store/src/lib.rs

```diff
@@ -995,259 +563,31 @@ fn handle_ft_worker_result(body: &[u8], context: &[u8]) -> anyhow::Result<()> {
 }
 
 fn handle_eth_sub_event(
-    our: &Address,
     state: &mut State,
     event: eth::SubscriptionResult,
-    requested_apis: &mut HashMap<PackageId, RequestedPackage>,
-) -> anyhow::Result<()> {
+) -> Result<(), AppStoreLogError> {
     let eth::SubscriptionResult::Log(log) = event else {
-        return Err(anyhow::anyhow!("got non-log event"));
-    };
-    state.ingest_listings_contract_event(our, *log, requested_apis)
-}
-
-fn fetch_package_manifest(package: &PackageId) -> anyhow::Result<Vec<kt::PackageManifestEntry>> {
-    let drive_path = format!("/{}/pkg", package);
-    Request::to(("our", "vfs", "distro", "sys"))
-        .body(serde_json::to_vec(&vfs::VfsRequest {
-            path: format!("{}/manifest.json", drive_path),
-            action: vfs::VfsAction::Read,
-        })?)
-        .send_and_await_response(5)??;
-    let Some(blob) = get_blob() else {
-        return Err(anyhow::anyhow!("no blob"));
+        return Ok(());
```

Comment:
This changed from an `Err` to an `Ok`. Just confirming: this was intended?

Reply:
good catch, made it an error